src/OrchardCore.Modules/OrchardCore.Tenants/TenantsConstants.cs

```diff
@@ -0,0 +1,6 @@
+namespace OrchardCore.Tenants;
+
+public class TenantsConstants
```

Comment:
I think it may be good idea to define something like this class at an upper level project like  `OrchardCore.Infrastructure.Abstractions`. This would allow us to reuse it anytime we want to split a string. Similar logic is defined in many places in the project

I don't know if `Abstractions` project is the best for this. But, it may not be a bad idea since it is used may most projects.

```C#
public class OrchardCoreConstants
{
    public const string StringSeperator = ", ";
}

public class OrchardCoreHelper
{
    public static string[] Split(string value)
    {
        return value?.Split(OrchardCoreConstants.StringSeperator, StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries)
            ?? Array.Empty<string>();
    }

    public static string Join(param string[] values)
    {
        if(values == null) 
        {
            return null;
        }

        return String.Join(OrchardCoreConstants.StringSeperator, values);
    }
}
```
@jtkech thoughts?

Reply:
Yes if the comma and space are often used as separators, maybe a `StringExtensions` and `StringArrayExtensions` but not sure, the good thing being to use the same static separators.

Okay for joining, but for splitting better to use a static array `new[] { ',', ' ' }` so that you can separate by comma with or whithout a following space(s). And in that case you don't need `TrimEntries`, only `RemoveEmptyEntries`.

Having the right split `Separators` and join `Separator`, the usage being.

    someString?.Split(Seperators, StringSplitOptions.RemoveEmptyEntries) ?? Array.Empty<string>();
    String.Join(Seperator, someArray ?? Array.Empty<string>();

I did this in ShellSettings but dedicated to the list of urlHosts and only for the splitting part.

Hmm, but why not having helpers just for the null checks, `OrchardCoreHelper` is more intended to be used in razor views, so maybe extensions and if often used maybe in `OC.Abstractions`, maybe under the `Modules` folder.

Reply:
@jtkech  in this case, we can't use `new[] { ',', ' ' }` since previously we could have used a space int he id. For example `this is an old name` in this case `new[] { ',', ' ' }` would break which is why we need to use `", "` I guess in this PR we don't need to add `OrchardCoreHelper` but maybe to use in other PR with  `new[] { ',', ' ' }` under a different name.

Reply:
Oops, okay I understand, but even if the comma separated list is not inputed by the user directly (if I understood correctly), can still be provided by a model in the ApiController, from a configuration source e.g. appsettings.json globally or for the AutoSetup, so I was thinking about the case for example if a comma is not followed by a space.

But maybe okay if the convention would be to use `, `.

Hmm, I think that in that case better to use a more dedicated separator, e.g. `|` and then we could split on `new[] { '|', ' ' }`, and document that we now can specify multiple profiles by separating them with `|`.

Sorry need to leave, will try to complete the review soon.

Reply:
@jtkech  Yes I understand. I think for consistency we should keep using a comma not a bar just because that is the command used separator. But if you want that change, we can do it. I'll wait your final review and we'll change anything needed. 

Currently everything is working as expect and is backward compatible.

Reply:
Anyway I was wrong, even if we use a `|` we still need to not split on space because of the old names, so my suggestion would be to split on `,` without space and then trim entries. About joining no matter so why not using this same separator. Then yes no need for an helper just to split / join on a single char, neither we need to use a const for a single char, only useful when we use an array.

> because that is the command used separator

Sorry I don't understand.

Reply:
@jtkech that is correct. Changing the separator in this case won't add any value. 

Let you me know you have any other requests or if you approve this PR

Reply:
What I meant is that you can split on a single char `','`, not a string having a space `", "`, e.g. if a source of data provides `"profile1,profile2"` in place of `"profile1, profile2"`.

Idem for joining by using the same single char `","` if it is just to persist the data as a comma separated string, still representing an array.

This everywhere and without defining a `TenantsConstants.FeatureProfileSeperator` const string.

    someString?.Split(',', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries)
        ?? Array.Empty<string>();

    String.Join(',', someArray ?? Array.Empty<string>());

Also, what you meant when saying: `because that is the command used separator`?

Reply:
> because that is the command used separator


There is a typo in there "because that is the common used separator"

I'll update the code when I get back to use a single comma instead of the constant.

Reply:
Okay cool then, for other things I trust @sebastienros which approved the PR.

Reply:
Yes I understand. I have not merged it because you had comments here so I want to make sure you are good with the changes.

Reply:
@jtkech I made the last changes. If I'll wait on merging this until tomorrow just in case you have any additional feedback.

thank you